partiql-planner/src/main/kotlin/org/partiql/planner/internal/astPasses/NormalizeSelect.kt

```diff
@@ -219,7 +212,8 @@ internal object NormalizeSelect {
                     args = tupleUnionArgs,
                     setq = null // setq = null for scalar fn
                 ),
-                setq = select.setq
+                setq = select.setq,
+                coercible = false
```

Comment:
What was coercible added for exactly?

Reply:
I removed this part to reduce noise in this PR.